website/views/options/settings/subscription.jade

```diff
@@ -49,7 +49,9 @@ script(id='partials/options.settings.subscription.html',type='text/ng-template',
 
         div(ng-if='user.purchased.plan.customerId')
           .btn.btn-primary(ng-if='!user.purchased.plan.dateTerminated && user.purchased.plan.paymentMethod=="Stripe"', ng-click='Payments.showStripeEdit()')=env.t('subUpdateCard')
-          .btn.btn-sm.btn-danger(ng-if='!user.purchased.plan.dateTerminated && user.purchased.plan.customerId !== "group-plan"', ng-click='Payments.cancelSubscription()')=env.t('cancelSub')
+          .btn.btn-sm.btn-danger(ng-if='user.purchased.plan.paymentMethod!="Apple" && user.purchased.plan.paymentMethod!="Google" && !user.purchased.plan.dateTerminated && user.purchased.plan.customerId !== "group-plan"', ng-click='Payments.cancelSubscription()')=env.t('cancelSub')
```

Comment:
Sure thing, I see now the constants already defined in server/libs. :)
I notice some other bits of logic lurking in this view, shall I tidy it all up while I'm at it?